docs/consume-packages/Package-References-in-Project-Files.md

```diff
@@ -386,3 +386,34 @@ You can control various behaviors of restore with lock file as described below:
 | `-LockedMode` | `--locked-mode` | RestoreLockedMode | Enables locked mode for restore. This is useful in CI/CD scenarios where you want repeatable builds.|   
 | `-ForceEvaluate` | `--force-evaluate` | RestoreForceEvaluate | This option is useful with packages with floating version defined in the project. By default, NuGet restore will not update the package version automatically upon each restore unless you run restore with this option. |
 | `-LockFilePath` | `--lock-file-path` | NuGetLockFilePath | Defines a custom lock file location for a project. By default, NuGet supports `packages.lock.json` at the root directory. If you have multiple projects in the same directory, NuGet supports project specific lock file `packages.<project_name>.lock.json` |
+
+### AssetTargetFallback
```

Comment:
Since this is a `###`, it doesn't appear in the "In this article" list. It's part of the "locking dependencies" section, which I don't think it right.

```suggestion
## AssetTargetFallback
```